tbx/core/blocks.py

```diff
@@ -428,6 +444,9 @@ class StoryBlock(blocks.StreamBlock):
     image = ImageBlock(
         template="patterns/molecules/streamfield/blocks/image_block.html",
     )
+    wide_image = WideImageBlock(
+        template="patterns/molecules/streamfield/blocks/wide_image_block.html",
+    )
```

Comment:
Please see my earlier comment regarding reuse of the `ImageBlock`.

```suggestion
    wide_image = ImageBlock(
        template="patterns/molecules/streamfield/blocks/wide_image_block.html",
    )
```

In the wagtail admin, this will be labelled as **Wide image**. If we need a different label, then we could pass a `label` keyword argument to the `ImageBlock`, just as we did for `template`.

---

If time allows, we could further enhance UX by grouping similar blocks using the `group` keyword argument.

reference: https://docs.wagtail.org/en/v5.2.3/reference/streamfield/blocks.html#block-options

Reply:
thank you! I will fix this now.

Reply:
New grouping in the admin:

![Screenshot 2024-02-28 at 08 45 51](https://github.com/torchbox/torchbox.com/assets/771869/a77570c9-1aac-4b3d-8929-cd42621c75b4)

Reply:
Updated in f40e404a010704cfa57a609f47027afc3e6b7e03